src/main/java/au/com/rayh/XCodeBuilder.java

```diff
@@ -246,11 +252,32 @@ public XCodeBuilder(Boolean buildIpa, Boolean generateArchive, Boolean cleanBefo
         this.interpretTargetAsRegEx = interpretTargetAsRegEx;
         this.ipaManifestPlistUrl = ipaManifestPlistUrl;
         this.ipaExportMethod = ipaExportMethod;
+        this.executeTests = executeTests;
+        this.skipBuild = skipBuild;
     }
 
     @Deprecated
     public XCodeBuilder(Boolean buildIpa, Boolean generateArchive, Boolean cleanBeforeBuild, Boolean cleanTestReports, String configuration,
                         String target, String sdk, String xcodeProjectPath, String xcodeProjectFile, String xcodebuildArguments,
+                        String cfBundleVersionValue, String cfBundleShortVersionStringValue, Boolean unlockKeychain,
+                        String keychainName, String keychainPath, String keychainPwd, String symRoot, String xcodeWorkspaceFile,
+                        String xcodeSchema, String buildDir, String developmentTeamName, String developmentTeamID, Boolean allowFailingBuildResults,
+                        String ipaName, Boolean provideApplicationVersion, String ipaOutputDirectory, Boolean changeBundleID, String bundleID,
+                        String bundleIDInfoPlistPath, String ipaManifestPlistUrl, Boolean interpretTargetAsRegEx, String ipaExportMethod) {
+        this(buildIpa, generateArchive, cleanBeforeBuild, cleanTestReports, configuration,
```

Comment:
I'm not sure from your comment whether you still want changes made in this PR or not. Could you clarify?